Sources/Parley/Views/ParleyComposeView/ParleyComposeView.swift

```diff
@@ -380,6 +390,17 @@ public class ParleyComposeView: UIView {
 
         present(alertController, animated: true, completion: nil)
     }
+    
+    private func uploadFile() {
+        let documentPicker: UIDocumentPickerViewController
+        if #available(iOS 14.0, *) {
+            documentPicker = UIDocumentPickerViewController(forOpeningContentTypes: [.pdf], asCopy: true)
+        } else {
+            documentPicker = UIDocumentPickerViewController(documentTypes: [String(kUTTypePDF)], in: .import)
```

Comment:
Do types based on ParleyMediaMimeType if possible (so that when adding a new supported mime type, it also adds here automatically?)

Reply:
Added static variables on ParleyMediaType and reference them here. Since not all media types are supported by the document picker it's not clear if this should be linked to the enum. However it does create safety for added cases to at least check if a new document type should be added.